lib/ueberauth/strategy/slack.ex

```diff
@@ -46,7 +46,13 @@ defmodule Ueberauth.Strategy.Slack do
   @doc false
   def handle_callback!(%Plug.Conn{ params: %{ "code" => code } } = conn) do
     module = option(conn, :oauth2_module)
-    token = apply(module, :get_token!, [[code: code]])
+    params = [code: code]
+    options = %{
```

Comment:
I think the redirect_uri can just go in the params. I don't think there's any reason why we need to put it int eh client options is there?